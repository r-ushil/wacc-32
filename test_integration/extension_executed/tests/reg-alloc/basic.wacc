# simple integer variable declaration

# Output:
# #empty#

# Program:

begin
  int a = 1;
  int b = 1;
  int c = 1;
  int d = 1;
  int e = 1;
  int f = 1;
  int g = 1;

  int x = a + b + c + d + e + f + g;

  # int x = 5;

  println x;

  skip
end
