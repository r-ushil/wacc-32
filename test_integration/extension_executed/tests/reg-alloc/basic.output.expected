Compiler exit code: 0
--------- stdout ---------
7
--------------------------
Exit code: 0
